docs/Configuration.md

```diff
@@ -898,6 +900,12 @@ class CustomEnvironment extends NodeEnvironment {
   runScript(script) {
     return super.runScript(script);
   }
+
+  handleTestEvent(event, state) {
```

Comment:
This wont work with NodeEnvironment. Current implementation is going to work only with jest-environment.

Reply:
I don't follow? This is `jest-environment-node`. Or do you mean running circus outside of jest?

Reply:
Sorry, I misunderstood the jestAdapterInit. Was quicker typing than thinking.

Reply:
Ah, ok. So we're good? :)

Reply:
yes :)